ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installation-nightly-4.18.yaml

```diff
@@ -637,6 +637,19 @@ tests:
     test:
     - ref: cucushift-installer-check-cluster-health
     workflow: cucushift-installer-rehearse-gcp-ipi-nested-virtualization-osdisk-type-size
+- as: gcp-ipi-arm-f28-ota
+  cron: 35 22 4 * *
+  steps:
+    cluster_profile: gcp-qe
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:arm64-latest
+    env:
+      COMPUTE_NODE_TYPE: t2a-standard-4
+      ENABLE_OTA_TEST: OCP-32747
```

Comment:
It's also needed to be enabled in 4.19?

Reply:
Actually, we need to enabled it from ocp 4.14 to 4.19